libs/boost/Makefile

```diff
@@ -12,18 +12,17 @@
 
 
 include $(TOPDIR)/rules.mk
-include $(INCLUDE_DIR)/nls.mk
 include $(INCLUDE_DIR)/target.mk 
 
 PKG_NAME:=boost
-PKG_VERSION:=1_61_0
+PKG_VERSION:=1.62.0
 PKG_RELEASE:=1
 
-PKG_SOURCE:=$(PKG_NAME)_$(PKG_VERSION).tar.gz
+PKG_SOURCE:=$(PKG_NAME)_1_62_0.tar.gz
 PKG_SOURCE_URL:=@SF/boost
-PKG_BUILD_DIR:=$(BUILD_DIR)/$(PKG_NAME)_$(PKG_VERSION)
-HOST_BUILD_DIR:=$(BUILD_DIR_HOST)/$(PKG_NAME)_$(PKG_VERSION)
-PKG_MD5SUM:=874805ba2e2ee415b1877ef3297bf8ad
+PKG_BUILD_DIR:=$(BUILD_DIR)/$(PKG_NAME)_1_62_0
+HOST_BUILD_DIR:=$(BUILD_DIR_HOST)/$(PKG_NAME)_1_62_0
+PKG_SHA256SUM:=440a59f8bc4023dbe6285c9998b0f7fa288468b889746b1ef00e8b36c559dce1
```

Comment:
There is no PKG_SHA256SUM tag. Use PKG_MD5SUM. The download code will inspect this tag to determine the length of the checksum and handle it appropriately as MD5 or SHA256 - really.